### ISSUE TITLE
Unable to import GKE multi-AZ zonal clusters

### ISSUE THREAD

**Issue by kkaempf** [rancher/dashboard#8256]
Internal ref: SURE-4599

Importing a local GKE-hosted cluster (v1.22.8-gke.200) with workers in multiple AZ fails in Rancher 2.5.12/2.6.x

Repro steps:

- Provision of a GKE cluster - v1.22.8-gke.200 (manually  from the GCP console or using Terraform), which  multiple availability zones for worker nodes
- Import this GKE cluster into Rancher

On investigation, when the UI gets the list of clusters (from GCP) for the given region its returning the location of the control plane and the locations that should be used by default for the nodes:

```
"location": "us-central1-c",            
"locations": [                
   "us-central1-b",
   "us-central1-c"            
]
```

**Comment by nwmac** [comment]
PR: https://github.com/rancher/ui/pull/4971

**Comment by cpinjani** [comment]
**Issue https://github.com/rancher/dashboard/issues/8256 - Test Results - :white_check_mark:** 

Issue reproduced on unfixed version:
![image](https://github.com/rancher/dashboard/assets/73099870/1b5fd0eb-c3b6-44b8-9caa-9f5e5902bd32)

Issue fixed, verfied on build `v2.7 - 0554da2`, able to import GKE multi-AZ zonal & regional clusters.
![image](https://github.com/rancher/dashboard/assets/73099870/b1f307f1-9927-4d40-a4cb-44d868ec782d)

GKE cluster:
![image](https://github.com/rancher/dashboard/assets/73099870/7a30f436-f2a3-4026-bdce-336223780991)
